src/lib.rs

```diff
@@ -187,6 +192,36 @@ impl Device {
         }
     }
 
+    pub fn usb_purge_tx_buffer(&mut self) -> Result<()> {
+        let result = unsafe { ffi::ftdi_usb_purge_tx_buffer(self.context) };
+        match result {
+            0 => Ok(()),
+            -1 /* read */ | -2 /* write */ => Err(Error::RequestFailed),
+            -3 => unreachable!("uninitialized context"),
+            _ => Err(Error::unknown(self.context)),
+        }
+    }
+
+    pub fn usb_set_event_char(&mut self, value: u8, enable: bool) -> Result<()> {
```

Comment:
Does `enable` mean that the value does not matter if `enable` is false?
If so, please make it `pub fn set_event_char(value: Option<u8>) -> Result<()>`

Reply:
I am not sure if this is the case. [Documentation](https://www.intra2net.com/en/developer/libftdi/documentation/ftdi_8c.html#a71eeabe2ee5474d5c0ceddc2e08766cd) is not detailed, but it looks like event and error char settings may be stateful for current context. In other words, it should be possible to set or clear them when needed.

Reply:
I've looked through the documentation and don't see any way to either read back that setting, or enable it without passing the new event character. Let's go with an `Option<u8>`, we can always add a lower level method if someone shows a situation where that actually matters.

Reply:
Done